CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
-### 0.4.3 (Next)
+### 0.5.0 (Next)
+
+* [#48](https://github.com/dblock/strava-ruby-client/pull/55): Removes `client.running_races` - [@simonneutert](https://github.com/simonneutert).
+* Your contribution here.
+
+### 0.4.3 (2022/03/17)
```

Comment:
We haven't released this, so merge the sections.

Put back "Your contribution here."

Reply:
done! 

we could merge https://github.com/dblock/strava-ruby-client/pull/56 in before releasing v0.5.0

Reply:
If you want to release 0.5.0 you should do that first and separately from this PR.  Right now this PR has a date for 0.5.0 which isn't a thing.

Reply:
is this now ready? if so I will update the changelog of #56 afterwards and v0.5.0 would be a thing 🌎